build/elasticsearch/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  elasticsearch:
+    image: ${WBS_ELASTICSEARCH_IMAGE_URL}:${WBS_ELASTICSEARCH_VERSION}
+    build:
+      context: .
+      args:
+        - ELASTICSEARCH_IMAGE_URL
+        - ELASTICSEARCH_PLUGIN_WIKIMEDIA_EXTRA
+        - ELASTICSEARCH_PLUGIN_WIKIMEDIA_HIGHLIGHTER
+      tags:
+        - ${WBS_ELASTICSEARCH_IMAGE_URL}:latest
```

Comment:
ATM we do not use `:latest` on docker hub. Would this end up on docker hub as well?

Reply:
Really good point. I can't remember why I added this, but I think I had to in the current setup for CI or something. Will loop back on that, I'd rather not have it here at all and don't think I'd need or want it on Dockerhub, and would like to avoid breaking out to manage tags in the Publish area if we don't need to...

Reply:
Yeah, this is the reason I had to add it: https://github.com/wmde/wikibase-release-pipeline/blob/73869322a22dce35a436ee89ccde4384f6ef75cf/.github/actions/pull-ghcr/action.yml#L36 ... Hmmm...

Reply:
This is not really solved, is it?

Reply:
You explained below. Sorry for the noise. I lost track in the parallel convos going on here. Closing again.

Reply:
As mentioned in the other comment the tagging configuration in the docker-compose files isn't currently applied in the build in CI, but is used locally. Once we sort out our publishing workflow this will likely change. That part is a WIP.